### ISSUE TITLE
Upgrade min-dom dependency to ^4.1.0

### ISSUE THREAD

**Issue by jfastnacht** [bpmn-io/diagram-js#768]
### What should we do?

Please upgrade the `min-dom` dependency to `^4.1.0`.

### Why should we do it?

I've been trying to achieve proper WebJar compatibility for `bpmn-js` for a while now, the problem reached all the way to [component-event](https://github.com/component/event/commit/cd64074ad614ac50777fab68adc958a7adf5434f), which did not have a proper license specified. Now that this is fixed and [min-dom](https://github.com/bpmn-io/min-dom/commit/aa48cbebec04ec4cfeb962230836ea7bd6f550fa) has been updated, I'm stuck at `diagram-js`, which is referencing `^4.0.2` of `min-dom` (which is referencing the old version of `component-event`).

From a legal perspective: you've got a dependency with a missing license text and unproperly stated license in the `package.json` in your dependency tree.

**Comment by nikku** [comment]
The semver range is ^, which means everything compatible with `^4.0.2`, which includes `v4.1.0`. What keeps you from simply using the latest version?

If this is a blocker for you, please consider contributing the version bump. Otherwise we'll do it eventually during regular house keeping.

**Comment by jfastnacht** [comment]
This is not urgent for me, because it's just a side project, so if it's part of the house keeping it is perfectly fine. Thanks again!

Yes, that's correct and probably an issue with Maven and its dependency resolution. I honestly don't know why, because the version range is specified correctly in the `pom.xml` of `diagram-js`, but it still resolves to the older version somehow (tested on two seperate systems). I have specified the version of `min-dom` explicitly already, before the changes to `component-event` were considered all the way up in the dependency tree of `bpmn-js`. Now that it is, I tried to reverse my explicit versions to specify only `bpmn-js` as a dependency and stumbled over the problem described above. I just wanted to report it instead of going ahead with my solution and ignoring it that the problem probably affects others as well.

My solution for anyone with the same problem:

```xml
    <dependency>
      <groupId>org.webjars.npm</groupId>
      <artifactId>min-dom</artifactId>
      <version>4.1.0</version>
      <scope>runtime</scope>
    </dependency>
    <dependency>
      <groupId>org.webjars.npm</groupId>
      <artifactId>bpmn-js</artifactId>
      <version>11.4.1</version>
      <scope>runtime</scope>
      <exclusions>
        <exclusion>
          <groupId>org.webjars.npm</groupId>
          <artifactId>min-dom</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```
